public/docs/ts/latest/guide/reactive-forms.jade

```diff
@@ -0,0 +1,387 @@
+  ## Setup
+  Create a new project folder (`app`) and follow the steps in [QuickStart](../quickstart.html).
+
+include ../_quickstart_repo
+
+.l-main-section
+a#create-comp
+:marked
+  ## Create a _reactive forms_ component
+  The app needs a place for the form. Make a new file called `hero-signup.component.ts` in the `app` directory. 
+  Import `Component` from `@angular/core` and `FormControl`, `FormGroup`, `FormBuilder`, and `FormArray` from `@angular/forms`:
+
++makeExample('reactive-forms/ts/app/hero-signup.component.ts', 'reactive-comp-imports','app/hero-signup.component.ts')(format=".")
+
+:marked
+  Here’s what each of them do:
+
+  - `Component`: Angular’s component decorator.
+
+  - `FormArray`: Lets you instantiate **FormArrays**.
+
+  - `FormBuilder`: Helps you create **FormControls**, **FormGroups**, and **FormArrays**.
+
+  - `FormControl`: Allows you to instantiate individual **FormControls**. Also tracks the value and validity of an individual `FormControl`.
+
+  - `FormGroup`: Allows you to group **FormControls**, **FormArrays**, and other **FormGroups**.
+
+  Below the import statements, use the `@Component` decorator to provide metadata to Angular:
+
++makeExample('reactive-forms/ts/app/hero-signup.component.ts', 'reactive-comp-metadata','app/hero-signup.component.ts (excerpt)')(format=".")
+
+:marked
+  The first line, `moduleId: module.id` lets you use [relative paths](./cookbook/component-relative-paths.html). 
+  The `selector` tells Angular where to render the form. And the `templateUrl` is 
+  the location of the HTML, or the contents of the form.
+
+  Next, create the component class with a simple form model by exporting a class called `HeroSignUpComponent`, 
+  which will hold  the form model. The class name, `HeroSignUpComponent`, is arbitrary. You can 
+  name it whatever you like. The example code on this page has numbers after each 
+  iteration, but you don't need those numbers. **<-----IS THIS NECESSARY? MAYBE I'M NOT DOING THIS EXACTLY RIGHT**
+
++makeExample('reactive-forms/ts/app/hero-signup-versions.component.ts', 'v1','app/hero-signup.component.ts (excerpt)')(format=".")
+
+:marked
+  Inside the `FormGroup` called `form`, each `new FormControl()` instantiates a `FormControl` for the form. 
+  This way is repetitive, but it helps illustrate an important point later.
+
+  The form model is only half of a reactive form. In order to make it available to the user, the app needs a template.
+
+.l-main-section
+a#create-template
+:marked
+  ## Create the template
+  Now create `hero-signup.component.html` with the following code and save it in the `app` directory.
+
++makeExample('reactive-forms/ts/app/hero-signup-2.component.html', 'basic-form','app/hero-signup.component.html')(format=".")
+
+:marked
+  On the `<form>` tag, `novalidate` prevents the browser from attempting any native validations 
+  and `[formGroup]="form"` lets Angular know that this is the form you’re referring to in the component class.
```

Comment:
Might it be more clear to name the form something other than form? Maybe "heroForm"? That way if the reader knows to set the [formGroup]=<name of the form> and doesn't think that the syntax is literally always [formGroup]="form".